manifests/init.pp

```diff
@@ -12,8 +12,11 @@
   $omd_version='1.20', #or omd-1.11.20140328
   $omd_release=latest,
 ) {
-  
-  
+
+  if( $::osfamily == 'redhat' and $facts['os']['release']['major'] == '7' ) {
```

Comment:
this assumes stringify_facts is false which is/was not the case by default for puppet3 (so this only works in very rare cases), and the check could just verify that the OS major release is >=6 (let's add epel to 6 too, I have it installed and can't say if that's absolutely required or not), not just strictly = 7. Can you please rework ?